libraries/tests/utest/spi_asynch/spi_master_asynch.cpp

```diff
@@ -152,7 +164,7 @@ TEST(SPI_Master_Asynchronous, short_tx_0_rx)
 {
     int rc;
     // Write a buffer of Short Transfer length.
-    rc = obj->transfer( tx_buf,SHORT_XFR,NULL,0, callback, -1);
+    rc = obj->transfer( (const uint8_t *) tx_buf, SHORT_XFR,  (uint8_t *) NULL, 0, callback, 255);
```

Comment:
Nit: the last argument, should be consistent with the rest and change it there or revert this one.